systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -106,11 +111,36 @@
 @Tag(CONNECT)
 @Tag(CONNECT_COMPONENTS)
 @SuppressWarnings({"checkstyle:ClassDataAbstractionCoupling"})
+@SuiteDoc(
+    description = @Desc("Verifies the deployment, manual rolling update, and undeployment of Kafka Connect components."),
+    beforeTestSteps = {
+        @Step(value = "Deploy uber operator across all namespaces, with custom configuration", expected = "Uber operator is deployed"),
+        @Step(value = "Deploy management Pod for accessing all other Pods", expected = "Management Pod is deployed")
```

Comment:
Is is scraper Pod? So maybe something like
```suggestion
        @Step(value = "Deploy scraper Pod for accessing all other Pods", expected = "Scraper Pod is deployed")
```